test/functional/tests/install/test_install_rpm.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from time import sleep
+import pytest
+
+from api.cas import installer
+from api.cas.cas_module import CasModule
+from core.test_run import TestRun
+from test_utils import os_utils
+
+# paths
+path_to_pckgen = TestRun.usr.working_dir + "/utils/pckgen"
+path_to_sources = TestRun.usr.working_dir + "/."
+path_to_packages = {TestRun.usr.working_dir} + "/packages/"
+# RPM commands and flags
+install_command = 'rpm --install'
+uninstall_command = 'rpm --erase'
+all_matches = '--allmatches'
+ignore_warns = '--replacefiles -- replacepkgs'
+
+
+@pytest.mark.remote_only()
+@pytest.mark.without_installation()
+def test_install_from_rpm():
+    """
+        title: Test for installing OpenCAS from RPM
+        description: |
+          Prepare RPM for OpenCAS;
+          install OpenCAS from RPM;
+          uninstall OpenCAS.
+        pass_criteria:
+          - OpenCAS installs successfully
+          - OpenCAS uninstalls successfully
+    """
+    with TestRun.step("Download and build OpenCAS."):
+        installer.copy_opencas()
+        installer.build_opencas()
+
+    with TestRun.step("Create OpenCAS's RPM."):
+        # Use package generator script directly
+        TestRun.executor.run_expect_success(f"{path_to_pckgen} {path_to_sources} rpm")
+
+    with TestRun.step("Get names of packages to install."):
+        packages = TestRun.executor.run(f"ls -1 {path_to_packages}").stdout.splitlines()
+        # For now there should be only 2 packages, if there would be more packages,
+        # we'll need to add more statements. Splitting packages makes installation easier
+        for package in packages:
+            if "modules" in package:
+                module = package
+            else:
+                program = package
```

Comment:
If you don't want to be dependent on number of packages, you can:
- list files with their absolute paths:
`ls -1 {path_to_packages}/*`
- create a list to install:
`packages_to_install = ' '.join(packages)`
- and then simply install:
`rpm -i {packages_to_install}`

And to remove them:
- obtain packages names:
```
for package in packages:
    concatenate output of 'rpm -qp --qf %\{NAME\} {package}'
```
- pass that to `rpm -e`